builtin/logical/ssh/path_config_ca.go

```diff
@@ -228,6 +228,17 @@ func (b *backend) pathConfigCAUpdate(req *logical.Request, data *framework.Field
 
 		return nil, err
 	}
+
+	if generateSigningKey {
+		response := &logical.Response{
+			Data: map[string]interface{}{
```

Comment:
I wonder if we should have same response both when `generateSigningKey` is `true` and when its `false`. Another thing to notice here is that the response format of the public key here is different than the output of `ssh/public_key` endpoint (HTTPRawBody in the latter).

@jefferai @briankassouf thoughts?